com.kms.cura.entity/src/com/kms/cura/entity/Entity.java

```diff
@@ -24,4 +25,7 @@ public String getId() {
 	public void setId(String id) {
 		this.id = id;
 	}
+	public int getType() {
+		return -1;
```

Comment:
Don't return meaningless value like this. Should make this method abstract instead.